src/common/components/facilitator/ride.tsx

```diff
@@ -196,6 +233,14 @@ class Ride extends Component<Props, State> {
       pickupTimeAndDate: this.state.pickupTimeAndDate.toISOString(),
     };
 
+    if(moment(rideFromState.pickupTimeAndDate).isBefore(moment.now())){
+      this.setState({
+        
+        updatingError: new Error('Invalid Date and time'),
+      });
+      return window.alert("You can't create ride in the past")
```

Comment:
Seeing how we're setting the error on the state, could we display the error in the form rather than having an alert? Like what we've done for the status.

Reply:
it is already displaying the state, the error message will be on the bottom of the form like all other errors, perhaps i will just get rid of the window alert

Reply:
realise need to account for when changing from confirmed to cancelled otherwise it will have an erorr for invalid time 
```
if(this.state.originalRideState?.status != 'CONFIRMED' && moment(rideFromState.pickupTimeAndDate).isBefore(moment.now())){
      this.setState({
        
        updatingError: new Error('Invalid Date and time'),
      });
```